content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/azure-saml-custom-attributes-img1.png" alt="azure-saml-custom-attributes-img1" height="50%" width="50%">
+
+
+![Image](/img/0b085a5aef05404e9ecdf52cb9088feb/azure-saml-hpe_ccs_attribute.png)
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/azure-saml-hpe_ccs_attribute.png" alt="azure-saml-hpe_ccs_attribute" height="50%" width="50%">
+
+
+* Click **Download** under Step 3 : Federation Metadata XML
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/azure-saml-federation-metadata-download.png" alt="azure-saml-federation-metadata-download" height="50%" width="50%">
+
+## Step 2: Configure GCLP for SAML Federation
+
+* Login to GLCP and select Manage
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/manage.png" alt="manage" height="50%" width="50%">
+
+* Select the Authentication tile
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs-authentication.png" alt="ccs_authentication" height="50%" width="50%">
+
+* Claim your domain for SAML
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs-claim-domain.png" alt="claim_domain" height="50%" width="50%">
+
+* Upload the _Federation Metadata XML_ file from the previous section.
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs-samle-azure-metadata-summry.png" alt="metadatasummary" height="70%" width="70%">
+
+* Apply the following configuration settings. These should match the First and Last Name settings you set above for Azure.
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs-saml-config-settings-summary.png" alt="saml-settings" height="70%" width="70%">
+
+* Create the recovery user per the instructions
+* Validate the settings are correct
+* Save and Finish the configruation.
+* If you get an error that the SAML configuraiton wasn't completed using the account with the @domain.com. You'll have to log back up and login with the SAML domain and go through the above configuration again.
+
+## Login to GLCP and Aruba Central using Azure AD
+
+* Once you've completed the above steps, login to central using your Azure AD email.
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs_login.png" alt="ccs_login" height="40%" width="40%"><img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs_login_saml.png" alt="ccs_login_saml" height="40%" width="40%">
+
+* If everything is working correctly, you should have logged into GLCP and Aruba Central is an option to Launch.
+
+## Using Azure AD MFA
+* By default, Azure AD enables MFA. However, for testing and demos, it's much easier to disable MFA on your accounts. To disable MFA, please see the following documentation: [What are security defaults](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/concept-fundamentals-security-defaults)
+
+## Troubleshooting
+* There's a useful 3rd party browser tool called: SAML Tracer
+* This tool will allow you to verify the attributes you're sending to Central.
```

Comment:
sending to Central (Is it Aruba Central or HPE GreenLake Central)? Please be specific here.